### ISSUE TITLE
Simplify temp botter removal

### ISSUE THREAD

**Issue by Autotonic** [GoelBiju/pinybot#55]
https://github.com/GoelBiju/pinybot/blob/pinybot-beta/pinybot.py#L1222

`py self.botters.remove(bot_user.nick)`

**Comment by GoelBiju** [comment]
You are completely right, the botter commands and the use of them should be improved by somehow enclosing all it's features possibly in a class to do what you suggested.

**Comment by GoelBiju** [comment]
I have removed the use of "botters" for the new version as @nortxort has put up a new level based system, this could be a better solution but is very similar to botters.
